classes/Cookie.php

```diff
@@ -590,6 +590,9 @@ public function getSession($sessionId)
         }
 
         if (!empty($session->getId())) {
+            // Update session date_upd
+            $session->save();
```

Comment:
strange having write operation in getter :( I suppose there was no other way?

Reply:
With object model, yes, but regarding session management, each time you retrieve it, you have to update the date where you get :sweat_smile: